package.json

```diff
@@ -18,7 +18,7 @@
     "templates:update-templates": "./update_templates.sh",
     "templates:build-templates": "node ./bin/bundle-templates.js templates dist/templates",
     "templates:build-arc-templates": "node ./bin/bundle-templates.js misc/arc-templates dist/arc-templates",
-    "templates:build-template-docs": "mdinject -w --root=./templates/. --docsroot=./templates/.. --sourceext=\".ts|.js|.html\" --targetext=\".md|.txt\"",
+    "templates:build-template-docs": "mdinject -w --root=./templates/. --docsroot=./templates/.. --sourceext=\".ts|.js|.html\" --targetext=\".md\"",
```

Comment:
Why `--docsroot=` is `./templates/..` and not `./templates/.`?
I think that's why `node_modules` got included. 

Also I feel like we should do `for` loop over all directories with templates – as otherwise code snippet names in different templates might clash.

Reply:
RE: dir scope:
Because we have READMEs nested above the src/ and assembly/ folders. 

RE: `for` loop: Probably.
1. We can change the lib to do that or add a flag to ignore node_modules.
2. We can use template specific snippet names. e.g. `hello-world-test-snippet`

Reply:
@pndpo hmm, README is still inside of folder for given template.

Reply:
Ah right. It should be like --root=templates/src/ && templates/assembly/ then --docsroot=templates/.

Reply:
> We can change the lib to do that or add a flag to ignore node_modules.

Or we can just do it in bash:

```
for template in templates/*; do mdinject -w --root=$template ...; done
```

Reply:
That will still look at node_modules inside of templates.

Reply:
It looks like it still works if I set the root flags to the ./templates without the need of a loop. Still want to ignore the node_modules though...

Reply:
@pndpo loop is needed to avoid name clashes, def not needed to work in general

Reply:
I realize that this should have been a different issue, but my repo with markdown injector now excludes node_modules by default.